### ISSUE TITLE
DependingPrices and Coupons

### ISSUE THREAD

**Issue by TripleNico** [smartstore/SmartStoreNET#2220]
Currently, we are in the process of setting up a campaign using coupons. In addition, we also have the DependingPrices plugin. During testing, we are finding out that unfortunately this is generating unwanted behavior.

The promotion we want to do is simple:
1. Customer receives coupon card in their order
2. Coupon can be used once per customer
3. Coupon only applies to orders over € xxx
4. Coupon adds 7.5% discount to the subtotal of the cart

No problems so far. However DependingPrices is also active and this creates a conflict. We want to apply either the coupon or DependingPrices. I.e. if the Coupon code is added DependingPrices logic should be ignored.

This obviously goes wrong because DependingPrices only works for a product page. There it adjusts the product price after which it is added to the cart. Then the Coupon is added to the subtotal of the cart and provides even more discount.

To put it simply, if a customer uses DependingPrices to get a 20% discount on a product and also uses the 7.5% coupon code, this gives a total discount of 27.5%. This is not our intention.

In short, is it possible to set that when a Coupon is used DependingPrices is not applicable?

**Comment by TripleNico** [comment]
Any news on this topic?
